tests/test_logging.py

```diff
@@ -0,0 +1,35 @@
+from pathlib import Path
+
+import pytest
+
+from unblob.logging import _format_message, noformat
+
+
+@pytest.mark.parametrize(
+    "value, expected",
+    (
+        (10, "0xa"),
+        (0xA, "0xa"),
+        ("10", "10"),
+        (noformat(10), 10),
+        (noformat(Path("/absolute/path")), Path("/absolute/path")),
+        (noformat(Path("relative/path")), Path("relative/path")),
+        ("/string/absolute/path", "/string/absolute/path"),
+        ("string/relative/path", "string/relative/path"),
+    ),
+)
+def test_format_message_dont_care_root_path(value, expected):
+    assert expected == _format_message(value, Path("dummy/path/does/not/matter"))
+
+
+@pytest.mark.parametrize(
+    "value, extract_root, expected",
+    (
+        (Path("/a/b/c"), "/", "a/b/c"),
+        (Path("/a/b/c"), "/a/b", "c"),
+        (Path("/a/b/c"), "", "/a/b/c"),
+        (Path("/a/b/c"), "qwe", "/a/b/c"),
```

Comment:
A test case is missing where `extract_root` is also absolute, but not relative to value.